tutorials/algorithms/05_qaoa.ipynb

```diff
@@ -15,11 +15,12 @@
    "metadata": {},
```

Comment:
<p>The link points to the old QAOA docs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>